ForageBuddy/NameReader.cs

```diff
@@ -41,20 +42,33 @@ private static Bitmap TransformImage(Image image, int width, int height)
 
             using (var graphics = Graphics.FromImage(destImage))
             {
-                graphics.CompositingMode = CompositingMode.SourceCopy;
-                graphics.CompositingQuality = CompositingQuality.GammaCorrected;
-                graphics.InterpolationMode = InterpolationMode.Default;
-                graphics.SmoothingMode = SmoothingMode.None;
-                graphics.PixelOffsetMode = PixelOffsetMode.None;
-
                 using (var wrapMode = new ImageAttributes())
                 {
-                    wrapMode.SetWrapMode(WrapMode.Tile);
+                    wrapMode.SetWrapMode(WrapMode.Clamp);
                     graphics.DrawImage(image, destRect, 0, 0, image.Width, image.Height, GraphicsUnit.Pixel, wrapMode);
                 }
             }
 
             return destImage;
         }
+
+        private static Bitmap ToBinaryImage(this Bitmap image)
+        {
+            Bitmap grayScale = new Bitmap(image.Width, image.Height);
```

Comment:
Should name be `blackAndWhite` ?

Reply:
Probably shouldn't exist at all, we can use the image being passed in via side effects, rather than allocating a new object.

Reply:
removed.